### PR TITLE
Add docs for using a base template

### PR DESCRIPTION
**Pull request by pheinrichs**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: doc/Alerting/Templates.md [hunk 1]
```diff
@@ -65,6 +65,47 @@ Placeholders can be used within the subjects for templates as well although $fau
 
 The Default Template is a 'one-size-fit-all'. We highly recommend defining your own templates for your rules to include more specific information.
 
+## Base Templates
+If you'd like to reuse a common template for your alerts follow below
+
+create a file in
+` resources/views/alerts/template.blade.php`
+Scaffold this file with the following
```
Comment (laf) [inline doc/Alerting/Templates.md]: Maybe `As an example, add the following:`

**Comment by pheinrichs** [comment]
Should I update the recommended folder path and / or give include a base template they can use and just ignore that file? 
My concern is that the path i chose is a bit generic and LibreNMS could use it in the future?

**Comment by laf** [comment]
I'd be happy to just have this template file in resources/view/alerts/ and exclude it from git changes or just change the dir to something like you've suggested.

**Comment by laf** [comment]
@pheinrichs If you can update this to how you want it to be we can merge for the tagged 1.42 release.

**Comment by laf** [comment]
I was under the impression we were still discussing what's best with regards to excluding the dir as it's a standard name we might use ourselves later?

**Pull request merged**